ui/app/models/role-ssh.js

```diff
@@ -40,6 +40,7 @@ const CA_FIELDS = [
   'defaultExtensions',
   'allowBareDomains',
   'allowSubdomains',
+  'allowEmptyPrincipals',
```

Comment:
added to CA roles and in the more options toggle
<img width="667" alt="Screenshot 2024-09-23 at 6 01 59 PM" src="https://github.com/user-attachments/assets/326b5d2e-9706-47b4-a054-e5d32ea82d78">